libs/remix-lib/src/execution/txRunnerWeb3.ts

```diff
@@ -47,6 +47,9 @@ export class TxRunnerWeb3 {
   }
 
   _sendTransaction (sendTx, tx, pass, callback) {
+    let currentDateTime = new Date();
+    const start = currentDateTime.getTime() / 1000
+    console.log('start', start)
```

Comment:
remove log

Reply:
i am using this log to track performances, i'd rather keep it for the moment